opencti-platform/opencti-front/lang/back/fr.json

```
@@ -202,6 +202,9 @@
   "DST Payload": "DST Payload",
   "DST port": "Port DST",
   "Due date": "Date d'échéance",
+  "Duplicate a CSV ingester": "Dupliquer un injecteur CSV",
```

```suggestion
  "Duplicate a CSV ingester": "Dupliquer un ingester CSV",
```
It's ingester everywhere else in the fr lang file